### ISSUE TITLE
[BUG]  Historie erst nach Neuöffnen der Akte aktuell

### ISSUE THREAD

**Issue by stm9x9** [jlawyerorg/j-lawyer-org#1041]
Wenn ich eine Änderung in einer Akte mache (Dokument hinzufügen, Etikett ändern ...) und dann in den Reiter Aktenhistorie gehe, stehen diese da noch nicht. Erst wenn ich komplett aus der Akte raus und wieder reingehe.

**Comment by j-dimension** [comment]
Die Hostorie wird am Server generiert, nicht am Client. Ansonsten würden bspw. Änderungen über die API (genutzt durch externe Systeme) nicht mitgeschrieben werden.

Ein Neuladen der Historie bei jeder Clientaktion schließe ich aus. 

Möglich wäre ein Knopf für eine manuelle Aktualisierung, els Enhancement.

**Comment by stm9x9** [comment]
Letzteres wäre schon hilfreich.

Verständnisfrage: wann bekomme ich als Anwender bzw der Client es mit, wenn an der Akte was per API geändert wurde? also zb eine neue Frist gesetzt, ein Dokument hinzugefügt ... Auch erst nach einem Neuladen der Akte?

**Comment by j-dimension** [comment]
> Auch erst nach einem Neuladen der Akte?

Ja. Bzw, wenn die WV/Frist bereits heute fällig ist, dann ist sie mit leichter Verzögerung auf dem Desktop zu sehen.
